gpiozero/input_devices.py

```diff
@@ -889,6 +925,16 @@ def distance(self):
         """
         return self.value * self._max_distance
 
+    @property
+    def value(self):
+        """
+        Returns a value between 0, indicating the reflector is either touching
+        the sensor or is sufficiently near that the sensor can't tell the
+        difference, and 1, indicating the reflector is at or beyond the
```

Comment:
This is the only place that "the reflector" is mentioned, and (IMHO) it makes "the reflector" sound like a specific thing (e.g. a bike reflector) rather than "the distance measured by the sensor" that the rest of the docstrings talk about.